ios/SANITY_CHECK_RESULTS.md

```
@@ -0,0 +1,264 @@
+    return envKey
+}
+// Fallback
+return "test_pro_key"
+#else
+// TODO: Keychain
+return nil
+#endif
+```
+
+**Xcode Scheme Setup (optional):**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)